### ISSUE TITLE
Computing prediction intervals of Exponential Smoothing models

### ISSUE THREAD

**Issue by amoralico** [hawkular/hawkular-datamining#83]
I'm not sure whether the way predcition/confidecne intervals is correct.
I think you must use the  formulas that appear [here](http://www.sciencedirect.com/science/article/pii/016920709090103I)
and [here](https://www.researchgate.net/publication/5141707_Prediction_intervals_for_exponential_smoothing_using_two_new_classes_of_state_space_models) 
(models ANN, AAN, AAA)

**Comment by pavolloffay** [comment]
There are more variants of these algorithms if I remember correctly. As a reference I used some literature from Rob J Hyndman and also forecast package from R.

**Comment by amoralico** [comment]
Yes of course. The general aproximation you use for prediction intervals :
`predictValue +- Z95% * sqrt(var(residuals))` 
is correct, but the variance (var) is different for each model (simple, double and triple).
The last link I suggested (from Rob J Hyndman) shows how to compute these variances analitically, for the models you have implemented in the library (ANN = Simple, AAN = Double, AAA = Triple). I guess they are the Holt-Winters additive models.
